appveyor.yml

```diff
@@ -0,0 +1,10 @@
+version: 0.1.{build}
```

Comment:
I don't think so, according to appveyor's [docs](https://www.appveyor.com/docs/build-configuration/#appveyoryml).